crypto/rand/drbg_ctr.c

```diff
@@ -323,31 +340,34 @@ int drbg_ctr_init(RAND_DRBG *drbg)
         return 0;
     case NID_aes_128_ctr:
         keylen = 16;
+        ctr->cipher = EVP_aes_128_ecb();
         break;
     case NID_aes_192_ctr:
         keylen = 24;
+        ctr->cipher = EVP_aes_192_ecb();
         break;
     case NID_aes_256_ctr:
         keylen = 32;
+        ctr->cipher = EVP_aes_256_ecb();
         break;
     }
 
     drbg->meth = &drbg_ctr_meth;
 
     ctr->keylen = keylen;
+    ctr->ctx = EVP_CIPHER_CTX_new();
```

Comment:
I don't see you free this context anywhere, and that might explain why Travis thinks this leaks...

Reply:
I was thinking I needed to free that and then forgot all about it. :)

Reply:
Should be fixed